app/controllers/public/index.js

```diff
@@ -135,11 +136,9 @@ export default class IndexController extends Controller {
         })
         .catch(async e => {
           console.error('Error while saving order', e);
-          try {
-            await Promise.allSettled((attendees ? attendees.toArray() : []).map(attendee => attendee.destroyRecord()));
-          } catch (error) {
-            console.error('Error while deleting attendees after order failure', error);
-          }
+          await RSVP.allSettled(attendees ? attendees.toArray() : []).then(attendee => attendee.destroyRecord(),
```

Comment:
There is no RSVP defined and there is no need to change from try catch to then catch